src/glommio_ct.rs

```diff
@@ -0,0 +1,66 @@
+use futures_task::{FutureObj, LocalSpawn, Spawn, SpawnError};
+use futures_util::future::LocalFutureObj;
+use glommio_crate::{LocalExecutor, LocalExecutorBuilder};
+use std::future::Future;
+use std::rc::Rc;
+use crate::{LocalSpawnHandle, SpawnHandle, InnerJh, JoinHandle};
+
+/// A simple glommio runtime builder
+#[derive(Debug)]
+pub struct GlommioCt {
```

Comment:
I feel the API is much nicer. Just wondering, what happens if someone creates 2 of these on the same thread? Should we prevent it?

Reply:
I think we can leave it as is for now, since if user want to create two, they have to explicitly call `GlommioCt::new(name, cpu)`